scenarios/action_recognition/02_training_hmdb.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<p>This needs a little more explanation what is going on and what these itemized items mean. Plus a comparison between the reported accuracy, and the accuracy we achieve would be great. Looks like we only get 58.5% accuracy versus the reported 74.5% ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
done